source/about-govwifi/connect-to-govwifi.html.erb

```diff
@@ -13,34 +13,34 @@ description: Use your public sector staff email address to connect devices to Go
       <main class="govuk-grid-column-two-thirds" role="main" id="main">
         <h1 class="govuk-heading-l">Connect a device to GovWifi</h1>
         <p class="govuk-body">
-          You need a <%= link_to "public sector staff email address",
+          You need a <%= link_to "public sector email address",
           "/support/check-organisation-email-address", class: "govuk-link" %> to sign up for a GovWifi account.
         </p>
         <p class="govuk-body">
-          If you do not have a public sector staff email address, you can <%= link_to "connect to GovWifi as a visitor",
+          If you do not have a public sector email address, you can <%= link_to "connect to GovWifi as a visitor",
           "/support/visitor-access-to-govwifi", class: "govuk-link" %>.
         </p>
         <div class="govuk-inset-text">
-          If you manage IT services in a public sector organisation and would like to make GovWifi available in your buildings, please review our <%= link_to "network administrator guidance", "https://docs.wifi.service.gov.uk/", class: "govuk-link" %>.
+          If you manage IT services in a public sector organisation and would like to make GovWifi available in your buildings, see how to <%= link_to "offer GovWifi", "https://docs.wifi.service.gov.uk/", class: "govuk-link" %>.
         </div>
         <h2 class="govuk-heading-m">1. Sign up for a GovWifi account</h2>
-        <p class="govuk-body">Create a GovWifi account by using your public sector staff email address to:</p>
+        <p class="govuk-body">You need to:</p>
         <ul class="govuk-list govuk-list--bullet">
-          <li>send a blank email to <%= link_to "signup@wifi.service.gov.uk", "mailto:signup@wifi.service.gov.uk", class: "govuk-link" %></li>
+          <li>send a blank email from your public sector email address to <%= link_to "signup@wifi.service.gov.uk", "mailto:signup@wifi.service.gov.uk", class: "govuk-link" %>:</li>
           <li>leave the subject line blank</li>
         </ul>
-        <p class="govuk-body">We will email your username and password confirmation in reply.</p>
-        <p class="govuk-body">If you do not receive the username and password confirmation e-mail within a few minutes of signing up, please check your spam/junk e-mail folder.</p>
+        <p class="govuk-body">We'll reply with your username and password.</p>
+        <p class="govuk-body">Check your spam or junk email folder if the email does not arrive after a few minutes.</p>
```

Comment:
Is it helpful to front-load 'if' clauses, like in the original, so people can know straight away if something applies to them?

Reply:
Yep can do

Reply:
Done